### PR TITLE
logging to rolled over files

### PR DESCRIPTION
**Pull request by michaelsembwever**
Instead of logging to the console, log to files that are rolled over daily.

### CODE REVIEW

File: src/packaging/resource/cassandra-reaper-cassandra-ssl.yaml [hunk 1]
```diff
@@ -46,8 +47,11 @@ logging:
     io.dropwizard: WARN
     org.eclipse.jetty: WARN
   appenders:
-    - type: console
-      logFormat: "%-6level [%d] [%t] %logger{5} - %msg %n"
+    - type: file
+      logFormat: "%-6level [%t] %logger{5} - %msg %n"
+      currentLogFilename: reaper.log
+      archivedLogFilenamePattern: reaper-%d.log.gz
+      archivedFileCount: 99
```
Comment (adejanovski) [inline src/packaging/resource/cassandra-reaper-cassandra-ssl.yaml]: Shouldn't we specify a maximum size to control when rolling over will occur ?
Isn't 99 kind of a big number ?

Comment (michaelsembwever) [reply src/packaging/resource/cassandra-reaper-cassandra-ssl.yaml]: this rollover is daily.

when using `%d` it's daily, when using `%i` and maxFileSize it becomes size limited.

i just went for the simpler daily rotation approach here.

**Review by adejanovski — DISMISSED** [review]
I'm not crazy about changing the default behavior, which was to output the logs in the console.
I'd be in favor of adding the file appender commented for now or at least to keep the console appender too.

**Comment by michaelsembwever** [comment]
> I'd be in favor of adding the file appender commented for now or at least to keep the console appender too.

@adejanovski 
the console appender is back, in addition to the file appender, but the console appender only emits the important stuff (WARN|ERROR|FATAL), so that whatever captures the console log also doesn't end up at 6Gb or more like we've witnessed. one small code change is added to ensure the application start message always appears.

**Comment by adejanovski** [comment]
We should do so that packaged installs create the `reaper.log` files in `/var/log/cassandra-reaper` instead of creating them in the current directory. Other than that, I'm ok with the changes.

**Comment by michaelsembwever** [comment]
> We should do so that packaged installs create the reaper.log files in /var/log/cassandra-reaper instead of creating them in the current directory.

Good point! I'll take a look into that…

**Pull request merged**